### DISCUSSION TITLE
Farm different plots on two computers with the same key

### DISCUSSION THREAD

**Discussion by leoliyuanduo** [Chia-Network/chia-blockchain#2339] [Q&A]
Does my chance I get a reward depend on the total number plots on them? Or it depends on one of them?

**Comment by winlqb** [comment]
我也想知道这个问题，2个电脑挖的地能不能在同一台计算机上使用。

**Reply by leoliyuanduo** [reply]
我正在研究这两个链接，感觉是可以的，分享给你
https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines
https://github.com/Chia-Network/chia-blockchain/wiki/Chia-Keys-Architecture
